src/main/webapp/app/exercises/shared/example-submission/example-submission-import/example-submission-import.component.ts

```diff
@@ -58,6 +58,10 @@ export class ExampleSubmissionImportComponent implements OnInit {
         this.performSearch(this.search, 300);
     }
 
+    get searchTermEntered() {
+        return !!(this.state?.searchTerm?.length && this.state.searchTerm.length > 0);
```

Comment:
Checking the attribute length already makes sure that it‘s at least 1, since the value 0 is considered falsy.